### PR TITLE
Article/serverless new dimension

### PR DESCRIPTION
**Pull request by RichardForshaw**
Serverless blog article

### CODE REVIEW

File: _posts/2019-09-19-serverless-adventures-in-a-new-dimension.markdown [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## The Next Compute Dimension
+
+![Next Dimension](/img/serverless/new_dimension.jpg "The Next Dimension")
+
+The next step-change in web-based software delivery is already here, and has been here for a few years. That change is serverless computing, but as with many new things, it is not fully understood.
+
+Why do I refer to it as the next dimension? Because there are many things that it brings that you can’t see without changing your perspective, along with a new way of thinking that is needed to unlock them.
+Total hardware abstraction
```
Comment (temyers) [inline _posts/2019-09-19-serverless-adventures-in-a-new-dimension.markdown]: This should be a heading?

`### Total hardware abstraction`

As an element of style, I'd capitalise headings.

File: _posts/2019-09-19-serverless-adventures-in-a-new-dimension.markdown [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+S3 and SQS you say? But they have been around for ages! Yes they have. And they are serverless. Do you need to manage any servers to use them?
+
+### Not a silver bullet
+As much as I am an advocate of serverless, I know that it is not the answer to everything. It is a great solution for applications which are event-driven with medium-to-high volume, sporadic and fairly short-lived requests. Which puts it at the forefront for web-based applications. For example A Cloud Guru built their entire Learning Management System using Serverless services, a business which now supports over a million worldwide users and whose infrastructure bill is only about AU$500/month.
+
+If you are running heavy batch-based workloads, then there is better technology for you to use. But this is also key: platform and architecture are part of your tooling choices, and having an understanding of your needs and the options available to you will surely put you on the path to becoming an ‘Elite Performer’ yourself.
+
+
```
Comment (Hamo69) [inline _posts/2019-09-19-serverless-adventures-in-a-new-dimension.markdown]: Is there a call to action at the end?

Comment (Hamo69) [reply _posts/2019-09-19-serverless-adventures-in-a-new-dimension.markdown]: contact-my@mechanicalrock.io

**Comment by RichardForshaw** [comment]
All updated @temyers @Hamo69

**Pull request merged**